### PR TITLE
Update contributing documentation

### PR DESCRIPTION
**Pull request by dblenkus**

### CODE REVIEW

File: docs/contributing.rst [hunk 1]
```diff
@@ -62,13 +46,9 @@ Prepare Resolwe for development::
 Preparing database
 ==================
 
-Create a ``resolwe`` database::
-
-    # Remove database if exists
-    dropdb resolwe
+Start Docker containers::
```
Comment (gregorjerse) [inline docs/contributing.rst]: Docker containers should be started from "tests" subdirectory. Or with command 

> docker-compose --file tests/docker-compose.yml up --detach

**Comment by gregorjerse** [comment]
While following instructions in  _contributing.rst_ the command 

in the section _Building documentation_ did not work. The reply was

[gregor@delcek resolwe]$ python setup.py build_sphinx

> /usr/lib/python3.8/distutils/dist.py:274: UserWarning: Unknown distribution option: 'use_scm_version'
>   warnings.warn(msg)
> usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
>    or: setup.py --help [cmd1 cmd2 ...]
>    or: setup.py --help-commands
>    or: setup.py cmd --help
> 
> error: invalid command 'build_sphinx'

File: docs/contributing.rst (hunk 1 above)
Comment (dblenkus) [reply docs/contributing.rst]: Thanks, I've moved `cd tests` few lines up.

**Comment by dblenkus** [comment]
@gregorjerse It works in my environment. Are you sure you've installed Resolwe by running `pip install -e .[docs,package,test]` (especially the `docs` part) as stated in the documentation?

```
~/gen/resolwe $ python setup.py build_sphinx
running build_sphinx
creating /Users/domen/gen/resolwe/build
creating /Users/domen/gen/resolwe/build/sphinx
creating /Users/domen/gen/resolwe/build/sphinx/doctrees
creating /Users/domen/gen/resolwe/build/sphinx/html
Running Sphinx v1.6.7
loading pickled environment... not yet created
loading intersphinx inventory from https://docs.python.org/3/objects.inv...
loading intersphinx inventory from https://docs.djangoproject.com/en/2.2/_objects/...
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 10 source files that are out of date
updating environment: 10 added, 0 changed, 0 removed
reading sources... [100%] ref
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] ref
generating indices... genindex py-modindex
highlighting module code... [100%] resolwe.utils
writing additional pages... search
copying images... [100%] images/proc_03_types.png
copying downloadable files... [100%] example/example/processes/all_fields.yml
copying static files... done
copying extra files... done
dumping search index in English (code: en) ... done
dumping object inventory... done
build succeeded.
```

**Comment by dblenkus** [comment]
@gregorjerse Have you solved the problem with building documentation?

**Pull request merged**